applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1306,11 +1308,17 @@ int unicast_client_stop(void)
 	return 0;
 }
 
+
+#define SDU_INTERVAL_US (10000u)
```

Comment:
I guess this is just for testing as we need to get the actual interval.

Reply:
Correct.